lib/Interface/c_interface.cpp

```diff
@@ -2094,3 +2094,124 @@ int vc_parseMemExpr(VC vc, const char* s, Expr* oquery, Expr* oasserts)
   }
   return 1;
 }
+
+void _vc_useSolver(VC vc, stp::UserDefinedFlags::SATSolvers solver)
+{
+  /* Helper method to encapsulate setting a solver */
+  stp::STP* stp_i = (stp::STP*)vc;
+  stp::STPMgr* b = stp_i->bm;
+  b->UserFlags.solver_to_use = solver;
+}
+
+bool _vc_isUsingSolver(VC vc, stp::UserDefinedFlags::SATSolvers solver)
+{
+  /* Helper method to encapsulate getting a solver */
+  stp::STP* stp_i = (stp::STP*)vc;
+  stp::STPMgr* b = stp_i->bm;
+  return b->UserFlags.solver_to_use == solver;
+}
+
+bool vc_supportsMinisat(__attribute__((unused)) VC vc)
+{
+  return 1;
```

Comment:
If you're using `<stdbool.h>` please use the constants `true` and `false` that it provides.

Reply:
Done!